stories/Button.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router';
+
+import { storiesOf, addDecorator } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { linkTo } from '@storybook/addon-links';
+import { withInfo } from '@storybook/addon-info';
+import { text, boolean, select } from '@storybook/addon-knobs';
+
+import Provider from '../.storybook/Provider';
+import Button from '../src/ui/components/Button'; // or should I use AddonBadge here instead?
+
+storiesOf('Button', module)
+	.addDecorator((story) => <Provider story={story()} />)
+	.addDecorator((story) => (
+		<MemoryRouter initialEntries={['/']}>{story()}</MemoryRouter>
+	))
+	.add(
+		'options',
+		withInfo(
+			`Toggle the different button options using "Knobs" (see right panel).
+			Note with the default buttonType (none), you won't see much difference.`,
+		)(() => {
+			const label = text('Label', 'Hello Button');
+
+			const btnTypeOptions = {
+				neutral: 'neutral',
+				light: 'light',
+				action: 'action',
+				cancel: 'cancel',
+				confirm: 'confirm',
+				alert: 'alert',
+				none: 'none',
+			};
+
+			const buttonType = select(
+				'buttonType',
+				btnTypeOptions,
+				'none',
```

Comment:
Is this setting the default to `none`? I think `neutral` would be a better default because it's more common. I'm not sure why `none` is the default type for `Button`.

Reply:
yeh, I was just mimicking what the component shows as the default. is this ok given that? or still update that here?

Reply:
I guess this might be moot.. since we are removing knobs (for now?)..